src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -97,11 +154,27 @@ public JsonNode readJsonFile(File jsonFile) throws CommandException {
     public JsonNode readPersonsArray(JsonNode jsonTree) throws CommandException {
         JsonNode personsArray = jsonTree.get("persons");
         if (personsArray == null || personsArray.size() == 0) {
-            throw new CommandException("The JSON File is empty.");
+            throw new CommandException(MESSAGE_EMPTY_JSON_FILE_FAILURE);
         }
         return personsArray;
     }
 
+    /**
+     * Creates the directory for the CSV file if it does not exist.
+     *
+     * @param csvFile The CSV file for which the directory needs to be created.
+     * @throws CommandException If directory fails to be created.
+     */
+    public void createCsvDirectory(File csvFile) throws CommandException {
```

Comment:
Good handling of Boundary Value Analysis! Glad to see it being applied here after reading up on it this week.